src/restricted-endpoints/replication/replication-endpoints/replication-endpoints.controller.ts

```diff
@@ -36,6 +37,72 @@ export class ReplicationEndpointsController {
     private documentFilter: BulkDocumentService,
   ) {}
 
+  /**
+   * return information on the whole CouchDB
+   */
+  @Get()
+  couchDBInfo() {
+    return this.couchdbService.get();
+  }
+
+  /**
+   * returns information about a single DB
+   * @param db
+   */
+  @Get([':db'])
+  dbInfo(@Param('db') db: string) {
+    return this.couchdbService.get(db);
+  }
+
+  /**
+   * Reads a local document (which are not synced)
+   * @param db
+   * @param docId
+   */
+  @Get(':db/_local/:docId')
+  getLocalDoc(@Param('db') db: string, @Param('docId') docId: string) {
+    return this.couchdbService.get(db, `_local/${docId}`);
+  }
+
+  /**
+   * Updated a local document
+   * @param db
+   * @param docId
+   * @param body
+   */
+  @Put(':db/_local/:docId')
+  putLocalDoc(
+    @Param('db') db: string,
+    @Param('docId') docId: string,
+    @Body() body,
+  ) {
+    return this.couchdbService.put(db, body);
+  }
+
+  /**
+   * Get changes for a set of revs
+   * @param db
+   * @param body
+   */
+  @Post(':db/_revs_diff')
+  revsDiff(@Param('db') db: string, @Body() body) {
+    return this.couchdbService.post(db, '_revs_diff', body);
+  }
+
+  /**
+   * Get the changes stream.
+   * The `include_docs` params is automatically set to false.
+   * @param db
+   * @param params
+   */
+  @Get(':db/_changes')
+  changes(@Param('db') db: string, @Query() params) {
+    return this.couchdbService.get(db, '_changes', {
+      ...params,
+      include_docs: false,
```

Comment:
Instead of overwriting include_docs to always be false, we could consider returning an error code and denying it for include_docs completely. (Client making the request is currently not getting back what they should be able to expect.)

But I like the simplicity of this and don't see a big disadvantage currently :+1: